file-formats/bestpractices.md

```diff
@@ -0,0 +1,23 @@
+# Best practices
```

Comment:
Ideally, all this would go directly into the file format description that you request via GET /file_format. Then people have it available in all places without needing to find a best practice document.

Reply:
if it's best practices for backend, we can indeed use this: https://docs.openeo.cloud/federation/backends/fileformats.html
I didn't notice we could have a description in our file format metadata. I'll try to put some more relevant info there as well, or should we do it at aggregator level?

Reply:
Both? If users directly connect to VITO it's useful to have specific details about your implementation. The aggregator may need slightly different descriptions as it seems that the covered implementations vary...

Reply:
Followed the suggestion and moved the content from here to the fileformats federation page. As it's something different than the federation agreement I created a new subsection.